src/jit/gentree.h

```diff
@@ -1026,6 +1026,8 @@ struct GenTree
                                                //               of the static field; in both of those cases, the constant
                                                //               is what gets flagged.
 
+#define GTF_ICON_NULLTHIS           0x01000000 // GT_CNS_INT -- null constant is the this in a call, ok to inline call
```

Comment:
Suggestion: Perhaps it's just me, but I found it difficult to parse and understand this comment. Would it be accurate to say:
```
// GT_CNS_INT -- This contant is a null that is the 'this' argument to a call; it is ok to inline this call.

Reply:
Ah, this isn't needed anymore, I'll just revert it.